infra/eligibility_screener/template/ecs.tf

```diff
@@ -28,6 +42,78 @@ resource "aws_security_group" "allow-screener-traffic" {
   }
 }
 
+resource "aws_security_group" "allow-lb-traffic" {
+  name        = "screener_load_balancer_sg"
+  description = "Allows load balancers to communicate with tasks"
+  vpc_id      = module.constants.vpc_id
+
+  ingress {
+    description      = "HTTP traffic from VPC"
```

Comment:
This is HTTP traffic from anywhere, right? Since it's a public load balancer.

Reply:
yes